### PR TITLE
Clean up Buildkite CI

### PR DESCRIPTION
**Pull request by HeinrichApfelmus**
### Overview

This pull request cleans up our Buildkite pipelines for the new environment.
- [x] add an explicit queue `cardano-wallet`
- [x] migrate the database files for the `api` benchmark
- [x] suspend macos steps as we do not have a macmini

### CODE REVIEW

**Review by paolino — APPROVED** [review]
should we call the queue HAL ?

File: .buildkite/bench-api.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p nix coreutils gnugrep gawk buildkite-agent
```
Comment (Unisay) [inline .buildkite/bench-api.sh:2]: 👍🏼

**Review by Unisay — APPROVED** [review]

**Comment by Unisay** [comment]
> should we call the queue HAL ?

I'd call it `cardano-wallet`

**Comment by HeinrichApfelmus** [comment]
> I'd call it cardano-wallet

👍 I'm going to add the `zurich.12` machine to the `cardano-wallet` queue.

**Pull request merged**